File: python/ex047.py
# Desenvolva um aplicativo que mostre na tela o resultado da expressão 500 +
# 450 + 400 + 350 + 300 + ... + 50 + 0

counter = 500
soma = 0

while counter >= 0:
    print(f"{counter} +", end=" ")
    soma += counter
    counter-=50
print(f"= {soma}")

File: csharp/ex047.cs
// Desenvolva um aplicativo que mostre na tela o resultado da expressão 500 +
// 450 + 400 + 350 + 300 + ... + 50 + 0

class Ex047
{
  public int counter = 500;
  public int soma = 0;

  public void FirstCountdown()
  {
    while (counter >= 0)
    {
      counter -= 50;
      soma += counter;
      Console.Write($" {counter} +");
    }
    Console.Write(soma);
  }
}

